server/threadpool.go

```diff
@@ -0,0 +1,61 @@
+/******************************************************************************
+ *
+ *  Description :
+ *    A very basic and naive implementation of thread pool.
+ *
+ *****************************************************************************/
+package main
```

Comment:
This is completely generic. Maybe move it to a separate package?

Reply:
Moved to `concurrency`.